Source/qol/statistics.cpp

```diff
@@ -0,0 +1,97 @@
+#include "qol/statistics.h"
+
+namespace devilution {
+
+Statistics myPlayerStatistics;
+std::map<std::string, std::string> statisticsFile;

```

Comment:
Globals starts with uppercase

Reply:
I was wondering shouldn't I just access those 2 by pointer placed in `Player` struct...

Reply:
@AJenbo Not quite on topic, but what led to this decision? In various C(++) projects, I hardly ever see globals starting with an uppercase letter, not to mention using the different case style from the rest of variables. Much more often they are just prefixed somehow (`g`, `g_`) and using the same case style as the members, locals, etc.